contracts/NationCred.sol

```diff
@@ -0,0 +1,43 @@
+//SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.17;
+
+import "./INationCred.sol";
+
+/**
+ * @notice Stores the passport IDs of active Nation3 citizens.
+ */
+contract NationCred is INationCred {
+    address public owner;
+
+    mapping(uint16 => bool) public activeCitizens;
+    uint16[] public passportIDs;
+
+    constructor() {
+        owner = address(msg.sender);
+    }
+
+    function setOwner(address owner_) public {
+        require(msg.sender == owner, "Only the owner can call this function");
+        owner = owner_;
+    }
+
+    function setActiveCitizens(uint16[] calldata passportIDs_) public {
```

Comment:
It took me a moment to see that the parameter name was different ad I was scratching my head as to what this function did, maybe just call it `activePassportIDs` or updated, or something to differentiate the variables more clearly.

Reply:
Good point, will fix! 👍

Reply:
Fixed:  https://github.com/nation3/nationcred-contracts/pull/45/commits/3e4c45958c89d34f0db0ff4d663e2d1b4c07404b